gradle.properties

```diff
@@ -1,8 +1,12 @@
 minecraft_version=1.12.2
 forge_version=14.23.5.2838
-mappings_version=snapshot_20180717
+mappings_version=stable_39
```

Comment:
I'd suggest matching BASE's mappings https://github.com/The-Acronym-Coders/BASE/blob/develop/1.12.0/gradle.properties#L3

Reply:
Why isn't BASE using stable39?